### DISCUSSION TITLE
Re-showing dismissed bottom sheet

### DISCUSSION THREAD

**Discussion by quinn305** [LucasMucGH/BottomSheet#90] [Q&A] [answered]
Hello, is there a way to re-show the bottom sheet once it has been dismissed? I can not find any such method to do that in the code. As such, I must conditionally display the View on an isDisplayed view attribute and then turn this off when OnGesture is triggered, i.e. somewhere in the main screen is tapped. However this gives a clunky appearance. Is there a way that a little arrow button would show up when the bottomSheet is dismissed, and you click it to make the sheet re-appear?

**Comment by lucaszischka** [comment, marked as answer]
No, there is no such button. But you can make your own button to show the BottomSheet yourself.
You only need to set the `bottomSheetPosition` of your BottomSheet to any position you want it to be:
```swift
Button {
    self.bottomSheetPosition = .relative(0.4)
} label: {
    Text("Open BottomSheet")
}
```
